### DISCUSSION TITLE
Making a client via Github build that only receives connections?

### DISCUSSION THREAD

**Discussion by thermafensterbau** [rustdesk/rustdesk#4178] [General]
We have the following scenario.
Actually we are evaluating Rustdesk. We have very low usage of doing remote sessions (only 3-5 times a week), but it will come very handy for us for various reasons to host a own server via Docker.
Everything runs fine so far, but we encountered the following issue:
We´d like to publish a Rustdesk client on our website, so customers who need some help, may download it and get support from us.
The thing is, when 2 people download the client from our website, this means they can do sessions using our servers.
Is there a way to prevent this?
My idea was, that there may be a stripped down "customer" client that only can receive connections.
May this be possible?
Other ideas?
Thanks a lot!

**Comment by rustdesk** [comment]
We have no plan to add it, but we accept this PR.

**Comment by JReally** [comment]
I'm no expert but when I include the public key in the client that I use and the remote user uses a client without the public key I can still connect to the remote user.  If the remote user attempts a connection (without the public key) they get an exception.

That would seem to be sufficient for your situation. 
I have not done extensive testing but it does seem to work that way.

JR

**Comment by perplexityjeff** [comment]
I know this is an older discussion however I'd like to +1 to this. 

In our case we'd like to use this internally for assisting employees however want to not allow the end user to use rustdesk for remoting thus only allowing receiving, this should always be performed by an employee from IT to offer assistance (in our case).

I'd like to say however that I do like the software in its current state even lacking this feature from a personal standpoint however we'll not be using it most likely in our company.
